next/schemas/collection/Project.ts

```diff
@@ -10,3 +11,17 @@ export const projectSchema = z.object({
 });
 
 export type Project = z.infer<typeof projectSchema>;
+
+
+// Changing it for us: removing slug, date, and blocks? Add signup url and link
+// What does blocks do im not sure, ask someone lol
+export const projectModalSchema = z.object({
+  title: z.string(),
+  image: z.any(),
+  description: z.string(),
+  signupurl: z.array(z.string()),
+  signuptext: z.array(z.string())
+});
+
+export type ProjectModalType = z.infer<typeof projectModalSchema>;
```

Comment:
No need to change this file actually! It's just to validate stuff coming from Strapi :)

Would recommend setting this file back just to avoid merge conflicts, and possibly pass it on in a Discord message to Chloe and Deiza cause it might help them.

Blocks are unused now I think.

Reply:
sweet i have changed this back!